### DISCUSSION TITLE
Monitor all threads, but separately

### DISCUSSION THREAD

**Discussion by lefterise** [VerySleepy/verysleepy#102] [Q&A] [answered]
I would like to make a capture that will profile all the threads of a process, but keep the statistics per thread separated.

The issue I have is that in my application some thread goes to 100%, When it does, I have a process that starts up very sleepy to monitor all threads. The aggregated data don't help me, I would like to be able to examine each thread separately despite capturing all threads

**Comment by CyberShadow** [comment, marked as answer]
> I would like to be able to examine each thread separately despite capturing all threads

Why not filter by thread in the results window, then? Very Sleepy already records which thread each sample is coming from.

**Reply by lefterise** [reply]
I have version 0.91, I just noticed the release notes mentioning that the feature is in the next release! nice!
